libs/core/avatar-group/avatar-group.component.ts

```diff
@@ -88,6 +94,10 @@ export class AvatarGroupComponent implements AvatarGroupHostConfig {
     @Input()
     size: AvatarGroupHostConfig['size'] = 'l';
 
+    /** Popover placement */
+    @Input()
+    popoverPlacement: Placement | null = null;
```

Comment:
use Nullable type.

Reply:
PopoverPlacement type is Placement | null. It's not Nullable<Placement>. If I change it I'm afraid it will broke already existing functionality